src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,125 @@
+package frc.robot.subsystems.climber;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonFX;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Ports;
+import frc.robot.subsystems.UnitModel;
+
+public class Climber extends SubsystemBase {
+    private final TalonFX master = new TalonFX(Ports.Climber.MASTER);
+    private final TalonFX slave = new TalonFX(Ports.Climber.SLAVE);
+    private final UnitModel unitModel = new UnitModel(Constants.Climber.TICKS_PER_METER);
+    private final Solenoid stopper = new Solenoid(Ports.Climber.STOPPER);
+    private final Solenoid gearboxPiston = new Solenoid(Ports.Climber.GEARBOX_PISTON);
+
+    public Climber() {
+        slave.follow(master);
+
+        master.setInverted(Ports.Climber.MASTER_INVERTED);
+        slave.setInverted(Ports.Climber.SLAVE_INVERTED);
+
+        master.setSensorPhase(Ports.Climber.MASTER_SENSOR_PHASE_INVERTED);
+        slave.setSensorPhase(Ports.Climber.SLAVE_SENSOR_PHASE_INVERTED);

```

Comment:
Does the slave have a sensor?

Reply:
Yes the talonFX have an internal sensor

Reply:
True, didn't think about it.
But are you using it? If not, don't configure it. You could maybe fuse the readings from the sensors, but I don't know how to feed the fused measurements back to the Talon FX, and I don't think it's necessary.